app/jobs/analytics/upsert_participant_profile_job.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class Analytics::UpsertParticipantProfileJob < ApplicationJob
```

Comment:
```suggestion
class Analytics::UpsertECFParticipantProfileJob < ApplicationJob
```
Really pedantic, but might be worth being explicit here. What do you think?